Connected/Assets/Scripts/Components/Battery.cs

```diff
@@ -3,9 +3,15 @@
 using UnityEngine;
 
 public class Battery : PowerSource {
+    [SerializeField]
+    float Voltage = 12f;
```

Comment:
Is there a special reason for breaking naming conventions with the capitalization here?

Reply:
Made it with capital letter since PowerSource has a public “get/set”-field “voltage”. I don’t know what came to my mind when creating this field, I’ll just make the PowerSource voltage into a protected field.